src/types/Votes.ts

```diff
@@ -0,0 +1,65 @@
+import { components } from 'src/api/types/generated-schema';
+
+type CohortPhase = string;
+
+type PhaseVotes = {
+  cohortPhase: CohortPhase;
+  votes: VoteSelection[];
+};
+
+export type DeleteProjectVotesRequestPayload = {
+  options: DeleteVoteSelection;
+};
+
+export type DeleteProjectVotesResponsePayload = components['schemas']['SimpleSuccessResponsePayload'];
+
+export type DeleteVoteSelection = {
+  projectId: number;
+  phase?: CohortPhase;
+  userId?: number;
+};
+
+export type GetProjectVotesResponsePayload = {
+  projectId: number;
```

Comment:
@joshbuchea can we also add projectName? BE PR was updated.

Reply:
You got it 👍

Reply:
@karthikbtf @tommylau523 should we also add `projectName` to the `UpsertProjectVotesResponsePayload`?

Reply:
Not needed, that is keyed off the projectId.
Project name is only for the client display purposes.

Reply:
Gotcha